packages/nimble/R/cppDefs_cppProject.R

```diff
@@ -228,8 +223,27 @@ cppProjectClass <- setRefClass('cppProjectClass',
                                        if(status != 0) 
                                            stop(structure(simpleError("Failed to create the shared library"),
                                                           class = c("SHLIBCreationError", "ShellError", "simpleError", "error", "condition")))
-                                       nimbleUserNamespace$sessionSpecificDll <- dyn.load(ssDllName, local = TRUE)
-                                   },                                  
+                                       return(dyn.load(ssDllName, local = TRUE))
+                                   },
+                                   compileDynamicRegistrations = function(showCompilerOutput = nimbleOptions('showCompilerOutput')) {
+                                       timeStamp <- format(Sys.time(), "%m_%d_%H_%M_%S")
+                                       dllName <- paste0("dynamicRegistrations_", timeStamp)
+                                       cppName <- paste0(dllName, ".cpp")
+                                       writeDynamicRegistrationsDotCpp(dynamicRegistrationsCppName, dynamicRegistrationsDllName)
+                                       nimbleUserNamespace$sessionSpecificDll <- compileStaticCode(dllName, cppName)
+                                   },
+                                   compileTensorflowWrapper = function(showCompilerOutput = nimbleOptions('showCompilerOutput')) {
+                                       # Load prerequisite DLLs.
+                                       library(tensorflow)
```

Comment:
better to use `if(!require(tensorflow)) stop('tensorflow package must be installed.  Try "install.packages(\"Rtensorflow\")"')` or similar.